src/pages/app-development/examples.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Adobe Commerce Extensibility Examples
+description: Learn how to use API Mesh, Adobe I/O Events, and App Builder to extend Adobe Commerce.
+---
+
+# Extensibility examples
+
+There are many ways to extend [Adobe Commerce](../index.md) today, such as [APIs](https://developer.adobe.com/commerce/webapi/) and [extensions](https://developer.adobe.com/commerce/marketplace/guides/sellers/extension-create/). However, these options in isolation only encapsulate a small fraction of possibilities with extensibility. This article speculates on what is possible with new and emergent offerings and more complicated integrations to explore the boundaries of what can be created in the future.
+
+## Understanding the components
+
+Adobe has several new and developing technologies that can further extend the Adobe Commerce platform. Each of the following services has unique features and capabilities, which are important to understand individually before considering more complicated examples.
+
+### Adobe Developer App Builder
+
+[App Builder](https://developer.adobe.com/app-builder/docs/overview/) is a complete framework that allows you to build and deploy custom web applications to extend Adobe Commerce and other Adobe solutions while running on Adobe infrastructure. With [React Spectrum](https://react-spectrum.adobe.com/react-spectrum/index.html) these single-page applications (SPAs) are only limited by your React knowledge.
+
+To learn more about App Builder, follow [this step-by-step tutorial](https://developer.adobe.com/app-builder/docs/getting_started/first_app/) on creating your first app. These instructions provide you with the foundational knowledge of what App Builder is and how it works.
+
+### API Mesh for Adobe Developer App Builder
+
+[API Mesh](https://developer.adobe.com/graphql-mesh-gateway/) is an Adobe service that allows you to take multiple disparate sources, such as GraphQL endpoints, REST APIs, JSON Schemas, and SOAP implementations, and combine them into a single queryable GraphQL endpoint. By creating a mesh, you can connect Adobe Commerce to App Builder, Adobe IO Gateway, and any other service that provides an accessible source. You can further customize your experience by [transforming](https://developer.adobe.com/graphql-mesh-gateway/gateway/transforms/) source data.
+
+![API Mesh diagram](../_images/api-mesh-diagram.png)
+<!-- [Link to .mmd file](../_images/api-mesh-diagram.mmd) -->
+
+API Mesh has several features which allow for further extensibility:
+
+- [Hooks](https://developer.adobe.com/graphql-mesh-gateway/gateway/hooks/) - Allow you to invoke composable local or remote functions, which can be used for authenticating or checking for a header before making a request.
```

Comment:
we only support Before ALL hooks - so that's something to pay attention to
Hooks again is not an industry standard word for extensibility - like for Adobe Commerce we had Plugins - react has hooks - which is the inspiration for this term
So I think a statement like
'Allow you to invoke composable local or remote functions, which can be used for authenticating or checking for a header before making a request.'
doesn't really explain hooks because it doesn't disambiguate the term
and composable local or remote functions - remote functions are not supported only in the TI version - and here we should only document the multitenant version
Also composable is a marketing term that doesn't resonate a lot with developers which is our target audience here - if you must reference it, do so by explaining what's composable about having a multitude of functions when reacting to an action or API event like beforeALL fetch of data

Reply:
@cpartica I was under the impression that remote functions were supported. It is covered in our public documentation: https://developer.adobe.com/graphql-mesh-gateway/gateway/hooks/#remote-composer-example. @revanth0212